src/common/components/SuggestionCard.jsx

```diff
@@ -8,38 +10,52 @@ export const SuggestionCard = ({
   username,
   profileImage,
 }) => {
+  const dispatch = useDispatch();
+
+  const handleFollowClick = () => {
+    dispatch(followUser(_id));
+  };
+
   return (
-    <Link
-      title={`@${username}`}
-      to={`/profile/${_id}`}
-      className="flex items-center p-2 pt-3 mb-4 rounded-lg shadow border dark:bg-gray-800 dark:border-gray-700"
-    >
-      {profileImage ? (
-        <img
-          loading="lazy"
-          src={profileImage.url}
-          alt={profileImage.original_filename}
-          className="w-10 h-10 mr-4 object-cover rounded-full flex-shrink-0 bg-gray-200"
-        />
-      ) : (
-        <div className="w-10 h-10 mr-4 text-lg flex items-center flex-shrink-0 justify-center font-semibold rounded-full bg-blue-500 text-white">
-          {firstName[0].toUpperCase()}
+    <div className="flex items-center p-2 pt-3 mb-4 rounded-lg shadow border dark:bg-gray-800 dark:border-gray-700">
+      <Link
+        title={`@${username}`}

```

Comment:
What is use of this title here?

Reply:
it provides more information about the tag, like a tooltip